### DISCUSSION TITLE
Progressbar

### DISCUSSION THREAD

**Discussion by MikeCZ23** [element-plus/element-plus#17782] [Q&A]
![Snímek obrazovky 2024-08-06 140615](https://github.com/user-attachments/assets/355f6c2a-6bf9-4f8c-81b9-2eccfa525fad)
how do I put it next to each other? I don't want it underneath me like this.

i have this code in file
```md
<div class="stavpr">
  <div class="infopr">Preklad</div>
  <div class="progpr"><el-progress :percentage="50" :stroke-width="15" striped /></div>
</div>
```
<hr>

![Snímek obrazovky 2024-08-06 140929](https://github.com/user-attachments/assets/2d2da783-fc0a-4b2b-a7d1-b7291be5b74a)
i want this

**Comment by HaceraI** [comment]
You can try display: flex;

```html
<div class="stavpr" style="display: flex; align-items: center; column-gap: 12px">
  <div class="infopr">Preklad</div>
  <div class="progpr" style="flex: 1"><el-progress :percentage="50" :stroke-width="15" striped /></div>
</div>
```

![image](https://github.com/user-attachments/assets/8ce7c549-ab6b-4d89-b5fd-641ea74e5427)

**Reply by MikeCZ23** [reply]
> You can try display: flex;
> 
> ```
> <div class="stavpr" style="display: flex; align-items: center; column-gap: 12px">
>   <div class="infopr">Preklad</div>
>   <div class="progpr" style="flex: 1"><el-progress :percentage="50" :stroke-width="15" striped /></div>
> </div>
> ```
> 
> ![image](https://private-user-images.githubusercontent.com/48341368/356456754-8ce7c549-ab6b-4d89-b5fd-641ea74e5427.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMxODQ3NDAsIm5iZiI6MTcyMzE4NDQ0MCwicGF0aCI6Ii80ODM0MTM2OC8zNTY0NTY3NTQtOGNlN2M1NDktYWI2Yi00ZDg5LWI1ZmQtNjQxZWE3NGU1NDI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA5VDA2MjA0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1MTJlZGUwNmZiOTk4ZDM3YzA5NjBmYzQyN2Q3Zjg2OWZhMDA1ODRhYWQ0Y2ZhMGZlNTk0MTQ3MmY5YjMyMjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GnxW9-tj323QFnwWAaAqyRv6k8_DRQYIePvR3Aeacdc)

This broke Blue part progressbar

**Reply by HaceraI** [reply]
I don't quite understand, I think the blue part of the progress bar is not broken, you can check that the width of the blue progress bar is half of the entire progress bar.
![image](https://github.com/user-attachments/assets/32759f0d-3c2b-4df2-920e-86666f488249)
![image](https://github.com/user-attachments/assets/7acb7aac-b480-4bb3-ac5f-a053176bbd1c)


Do you want the blue progress bar to start from the far left? (red)
![image](https://github.com/user-attachments/assets/4dd10333-36a3-4992-b4a3-a536ed512896)

**Reply by MikeCZ23** [reply]
I've already solved it with your instructions thanks

can I move these divs?
left:15px;?

**Reply by HaceraI** [reply]
Yes, CSS is very powerful. If you want to increase the spacing, you can try margin-left or padding-left, but for flex layout, I recommend using [column or row]-gap. You can find what you want here: https://developer.mozilla.org/en-US/docs/Web/CSS/margin-left
